data-structures-stacks/get-top.js

```diff
@@ -0,0 +1,8 @@
+
+export default function getTop(stack) {
+  if (stack.length === 0) {
```

Comment:
This check also isn't needed since `peek` returns undefined if the stack is empty.

You are actually lucky it isn't needed, since `stack.length` doesn't exist!